src/base/moduledownloader.lua

```diff
@@ -60,7 +60,9 @@
 		end
 
 		-- Unzip the module, and delete the temporary zip file.
-		verbosef(' UNZIP   : %s', destination)
+		if not p.action.isHeadless() then
```

Comment:
verbosef only prints when --verbose is specified... I think this isn't really needed, or the ```if not headless``` could be moved to here (https://github.com/premake/premake-core/blob/master/src/base/_foundation.lua#L402) maybe?

Reply:
It *could* be added there, but if the user uses `verbosef()` to print output in their headless action, it wouldn't function correctly.

The reason I added a check for that specific verbosef() call was primarily a safety measure, as I intended to use a headless action to pipe project metadata to a VSCode extension to provide a Visual Studio like configuration / platform switcher, and wanted to ensure that the premake output would be valid json, without extra information being written to stdout.